src/Events.php

```diff
@@ -51,6 +51,11 @@
     const MANDATE_ADDED_EVENT = 'mandate.added';
 
     /**
+     * A mandate has been edited, expects a DonorEvent
+     */
+    const MANDATE_EDITED_EVENT = 'mandate.edited';
```

Comment:
Please pull the last changes from master. There is a new naming convention here, this needs to be

```php
const MANDATE_EDITED_EVENT = 'MANDATE_EDITED_EVENT';
```